filter/filter.go

```diff
@@ -478,6 +479,33 @@ func (c *Client) GetDimensionOptionsBatchProcess(ctx context.Context, userAuthTo
 	return eTag, batch.ProcessInConcurrentBatches(batchGetter, batchProcessor, batchSize, maxWorkers)
 }
 
+// DeleteDimensionOptions completely removes the options array from a given dimension
```

Comment:
Can we not use named return arguments unless there's a need to (manipulating return arguments in deferred functions)?

Reply:
Another copy and paste... 
Let me change that.

Reply:
Done.